File: src/ExamRegister.WebApi.Abstractions/DTO/AuthorizationEntradaDTO.cs
﻿
namespace ExamRegister.WebApi.Abstractions.DTO {
    public partial class AuthorizationEntradaDTO {
        public string grant_type { get; set; }
        public string client_id { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        public string client_secret { get; set; }
        public string refresh_token { get; set; }        
    }
}

File: src/ExamRegister.UI/src/app/shared/DTOs/authorizationentrada-dto.ts
export class AuthorizationEntradaDTO {
  public grant_type: string | null = null;
  public client_id: string | null = null;
  public username: string | null = null;
  public password: string | null = null;
  public client_secret: string | null = null;
  public refresh_token: string | null = null;
}
